app/navigation/AuthorisedOfficialsNavigator.scala

```diff
@@ -39,6 +39,12 @@ class AuthorisedOfficialsNavigator @Inject()(implicit frontendAppConfig: Fronten
       case _ =>  routes.SessionExpiredController.onPageLoad()
     }
 
+    case IsAuthorisedOfficialPositionPage(index) => userAnswers: UserAnswers => userAnswers.get(IsAuthorisedOfficialPositionPage(index)) match {
+      case Some(true) => routes.IndexController.onPageLoad() // TODO redirect to next page once created
```

Comment:
can we combine this like some(_) with dead end controller? with to do comment

Reply:
Updated as per comment